content/cloud/changelog/spin-cloud-v0-6-0.md

```diff
@@ -0,0 +1,34 @@
+title = "Spin Cloud v0.6 Updates"
+template = "changelog_item"
+date = "2024-1-30T12:00:00Z"
+enable_shortcodes = true
+tags = ["spin_cloud_plugin"]
+[extra]
+type= "changelog_post"
+twitter_card_type = "summary_large_image" 
+image = "/static/image/changelog/TODO.jpg" 
+
+---
+
+Today we're announcing the minor release of [`spin cloud` plugin v0.6.0](https://github.com/fermyon/cloud-plugin), which allows the management of key value stores via CLI. 
+
+Highlights include:
```

Comment:
In case we want to be more verbose:
```suggestion
Before the v0.7.0 release of the [spin cloud plugin](https://github.com/fermyon/cloud-plugin), Spin applications could only use a "default" key value store. This release of the cloud plugin brings support for sharing key value stores between applications, linking multiple key values stores to an application, and dynamically updating which key value stores are linked to an application. The updated CLI includes all the commands needed to take these management actions:

`spin cloud kv create` will create a key value store on Fermyon Cloud
`spin cloud kv delete` will delete a key value store
`spin cloud kv rename` will rename a key value store
`spin cloud kv set` will set key value pairs in a key value store
`spin cloud link kv` will connect your key value store to a Spin application
`spin cloud unlink kv` will disconnect your key value store from a Spin application
`spin cloud deploy` will now detect whether your application uses key value stores and prompt you to create new key value stores or link the application to existing ones.
```

Reply:
We deleted the changelog from this PR as it was already covered in https://github.com/fermyon/developer/pull/1161